pages/homepage/[postId]/index.js

```diff
@@ -62,7 +70,7 @@ export default function Post({post}) {
             setComments([...comments]);
             setIsLoadingComments(false);
         })();
-    }, [post.postId]);
+    }, [post.postId, pinSuccess, unpinSuccess]);
```

Comment:
This is kinda scuffed, but ok. you could sorta give a signal to that one comment to add the pin or something. using events. but whatever.